docs/training/admin_guides.md

```diff
@@ -51,6 +49,10 @@ access for that geographic area as well as all its children-geographies. For
 example, in the above tree a user with access to California also has access to
 Marin County, Los Angeles County, Sausalito, San Rafael, and Los Angeles.
 
+#### Updating locations in VA Explorer
+
+When VAs are imported into VA Explorer, they are matched exactly on the locations loaded into the system in this step. If a VA does not have a valid location field, VA Explorer will track that mismatch as an error that either needs to be corrected in the VA Explorer locations file or in the underlying VA data. To add a location to VA Explorer, re-upload a revised location file following the `load_locations` management command.  If a row is deleted from the locations file, it will also be kept in VA Explorer and marked inactive.  To permanently delete locations in VA Explorer, re-upload a revised location file following the `load_locations` management command with the `--delete_previous` flag. Warning: doing so may delete all VAs in the database, so make sure to backup the system first.
```

Comment:
```suggestion
When VAs are imported into VA Explorer, they are matched exactly on the locations loaded into the system in this step. If a VA does not have a valid location field, VA Explorer will track that mismatch as an error that either needs to be corrected in the VA Explorer locations file or in the underlying VA data. To add a location to VA Explorer, re-upload a revised location file following the `load_locations` management command.  If a row is deleted from the locations file, it will also be kept in VA Explorer and marked inactive.  To permanently delete locations in VA Explorer, re-upload a revised location file following the `load_locations` management command with `--delete_previous=True`. Warning: doing so may delete all VAs in the database, so make sure to backup the system first.
```

Reply:
Addressed by [a1337ee](https://github.com/VA-Explorer/va_explorer/pull/288/commits/a1337ee246a631ccfc2c28936dc216208278f701)